db/init.sql

```diff
@@ -0,0 +1,150 @@
+DROP TABLE IF EXISTS "competition" CASCADE;
+DROP TABLE IF EXISTS "game" CASCADE;
+DROP TABLE IF EXISTS "invitation" CASCADE;
+DROP TABLE IF EXISTS "registration" CASCADE;
+DROP TABLE IF EXISTS "team" CASCADE;
+DROP TABLE IF EXISTS "user" CASCADE;
+
+CREATE TYPE shirt_size_enum AS ENUM (
+    'S', 'M', 'L', 'XL', 'XXL'
+);
+
+CREATE TYPE pizza_choice_enum AS ENUM (
+    'Cheese', 'Pepperoni', 'Bacon'
+);
+
+CREATE TYPE client_language_enum AS ENUM (
+    'cpp', 'js', 'lua', 'py', 'cs', 'java'
+);
+
+CREATE TYPE invitation_response_enum AS ENUM (
+    'accepted', 'declined'
+);
+
+CREATE TABLE competition
+(
+  id serial NOT NULL,
+  name character varying(64) NOT NULL,
+  cost integer,
+  description character varying(64),
+  image bytea,
+  start_time timestamp without time zone,
+  end_time timestamp without time zone,
+  is_open boolean,
+  is_running boolean,
+  min_num_team_members integer,
+  max_num_team_members integer, --
+  CONSTRAINT competition_pk PRIMARY KEY (id),
+  CONSTRAINT competition_name_is_unique UNIQUE (name)
+);
+
+
+CREATE TABLE team
+(
+  id serial NOT NULL,
+  name character varying(64) NOT NULL,
+  gitlab_id integer,
+  language client_language_enum,
+  is_paid boolean,
+  time_paid timestamp without time zone,
+  created timestamp without time zone,
+  is_eligible_to_win boolean,
+  is_embargoed boolean,
+  embargo_reason character varying(64),
```

Comment:
default to "Initial build not performed" a team is initially embargoed because an initial build has not been completed